### ISSUE TITLE
Feature Request: Toggle to view only favourites in a folder

### ISSUE THREAD

**Issue by Lppsoeht** [FossifyOrg/Gallery#230]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Gallery/issues) or [closed](https://github.com/FossifyOrg/Gallery/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Gallery/discussions) or [closed](https://github.com/FossifyOrg/Gallery/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Gallery/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

A toggle like "temporarily show hidden" (when in a folder) that basically hides every media that's not set as favourite, so it shows only favourites that are in the folder.

### Why do you want this feature?

It'll help filter medias, memories, a little more

**Comment by gemoshi07** [comment]
I would also like this feature that can show only the hidden files, by sorting the files or like a toggle.

**Comment by gemoshi07** [comment]
@Aga-C I wrote it here because I thought that I wouldn't want to make a new request with something similar. Should I make another post then?

**Comment by Aga-C** [comment]
@gemoshi07 Yes.

**Comment by zambuttus** [comment]
Yes, please! Marking Favorites is not that useful when you are only able to display them all together as a separate album. If I want to only show my faved pictures of my last vacation (which have their own album / folder), I don't want to first scroll through all faved pictures of work/home/...
